_posts/2025-09-10_community-survey-findings.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: "Vega Strike 2024 Community Survey Results"
+description: "Vega Strike Community Survey - 2024 Survey Results"
+author: BenjamenMeyer
+date: 2025-09-11 00:00:00 +0000
+categories: announcements community
+excerpt_separator: <!--more-->
+---
+# Overview
+In October 2024 the Vega Strike Development team put out a [survey](https://www.vega-strike.org/blog/2024/10/30/community-survey/); and
+today we'd like to share the results for the 53 responses we received. While not a lot of responses, it will be used to help inform
+the team. We'll repeat similar surveys in the future.
+
+The primary purpose of this survey was to help us target our CI system for what builds we focus on and what systems we make releases for.
+Please keep in mind that the questions allowed people to make more than one selection; so the totals between different sections may add up to more than the answers received
+and not match other sections.
+
+## Platform: Operating Systems
+
+We have been building on a lot of platforms as we were (a) guessing where our community might be, (b) trying to target popular platforms, and (c) covering platforms the
+current development team uses themselves. This survey provides us with the first bit of data to refine those based on your feedback.
+
+Results:
+```mermaid
+xychart-beta horizontal
+  title "Platform Usage"
+  x-axis "Platforms" ["Windows 11", "Windows 10", "Windows 8", "Windows 7", "Windows XP", "Mac OSX 15", "Mac OSX 14", "Mac OSX 13", "Mac OSX 12", "Red Hat Linux 9", "Red Hat Linux 8", "OpenSuSE Leap", "Linux Mint 21", "Linux Mint 22", "Ubuntu 20.04", "Ubuntu 22.04", "Ubuntu 24.04", "Fedora Linux", "Arch Linux", "Rocky Linux 9", "Rocky Linux 8", "Debian Bookworm", "Debian Bullseye", "Linux - Snap Packages", "Linux - FlatPak", "Gentoo Linux", "PC Linux OS", "Debian Trixie-Forkie", "Mageia LInux", "Slackware 15", "Slackware"]
+  y-axis "Users" 0 --> 26
+  bar [25,17,2,2,1,1,0,0,0,0,0,0,4,5,3,5,5,3,9,0,0,8,4,0,5,1,1,1,1,1,1]
+  line [25,17,2,2,1,1,0,0,0,0,0,0,4,5,3,5,5,3,9,0,0,8,4,0,5,1,1,1,1,1,1]
+
+```
+
+From this we can see that the majority is currently running on Windows (47 users across the various versions), followed by Linux users across a few different distros (57 users
```

Comment:
47 users followed by 57? Also 9+11+3+13+5+1+2+9 = 53.

Reply:
here's the source data
<img width="768" height="692" alt="image" src="https://github.com/user-attachments/assets/68bb9461-4927-4d4f-aea0-34118f89f1f7" />

I might have gotten some of the numbers wrong...I'll review. Just added it up again from the image and it matches 57.

Reply:
okay - found it - I left off 2 distros and mis-counted 1 by 2. I'll update it.